src/gui/src/settings/options-window.cpp

```diff
@@ -1055,6 +1069,9 @@ void OptionsWindow::save()
 	settings->setValue("imagePositionVideoH", positionsH.at(ui->comboImagePositionVideoH->currentIndex()));
 	settings->setValue("imageBackgroundColor", ui->lineImageBackgroundColor->text());
 
+	settings->setValue("actionButtons", ui->comboActionButtons->currentIndex());
```

Comment:
Since you bothered to  set the item data above, you can just use `currentData()` to get the enum value directly. It's a QVariant so you might need to cast to to an int with `toInt`.